### PR TITLE
Introduce a config to update console & myaccount callback url & access url

### PR DESCRIPTION
**Pull request by UdeshAthukorala**
### Proposed changes in this pull request

- Introduce a toml config to update console & myaccount callback url & access urls

  #### New Toml Configs

  ```
  [console]
  callback_url = "https://localhost:9001/t/{TENANT_DOMAIN}/console"
  access_url = "https://localhost:9001/t/{TENANT_DOMAIN}/console"
  
  [myaccount]
  callback_url = "https://localhost:9000/t/{TENANT_DOMAIN}/myaccount"
  access_url = "https://localhost:9000/t/{TENANT_DOMAIN}/myaccount"
  ```

### Related Issue

- https://github.com/wso2/product-is/issues/18164

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtUtil.java [hunk 1]
```diff
@@ -1084,6 +1084,28 @@ public static boolean isConsoleOrMyAccount(String name) {
         return ApplicationConstants.CONSOLE_APPLICATION_NAME.equals(name) ||
                 ApplicationConstants.MY_ACCOUNT_APPLICATION_NAME.equals(name);
```
Comment (madurangasiriwardena) [inline components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtUtil.java:1085]:
```suggestion
        return isConsole(name) || isMyAccount(name);
```

Comment (UdeshAthukorala) [reply components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtUtil.java]: Addressed with https://github.com/wso2/carbon-identity-framework/pull/5304/commits/a88bf6adea4698816742f7400abe77a68795a33b

**Review by Thisara-Welmilla — APPROVED** [review]

**Comment by Thisara-Welmilla** [comment]
Shall we check the `tenant_context.require_super_tenant_in_urls` config and based on that add `/t/carbon.super` in path.

**Review by Thisara-Welmilla — CHANGES_REQUESTED** [review]
Shall we check the `tenant_context.require_super_tenant_in_urls` config and based on that add `/t/carbon.super` in path.

**Comment by UdeshAthukorala** [comment]
> Shall we check the `tenant_context.require_super_tenant_in_urls` config and based on that add `/t/carbon.super` in path.

Addressed with https://github.com/wso2/carbon-identity-framework/pull/5304/commits/7b8bd9e4e79c57caa971e38fd823482904e4c13c

**Review by Thisara-Welmilla — APPROVED** [review]

**Pull request merged**